src/server/CookingApp/Controllers/CostCalculatorController.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace CookingApp.Controllers
+{
+    using CookingApp.Services.CostCalculation;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Options;
+    using MongoDB.Bson;
+    using System.Text.Json;
+
+    [ApiController]
+    [AllowAnonymous]
```

Comment:
On the 'dev' branch there mustn't be any [AllowAnonymous] endpoint nor controller. The structure of our API is designed for authenticated users only.